docker-compose/nginx/conf.d/monitoring.conf

```diff
@@ -0,0 +1,13 @@
+server {
```

Comment:
This needs to be addressed but let's not waste time with it for the moment, zabbix is going to alert anyway if this breaks so there is no way we can forget about it.